### PR TITLE
[PRM]  LRSD-4189 The fields never erase

### PR DESCRIPTION
**Pull request by eduardo-diniz**
[Ticket](https://liferay.atlassian.net/browse/LRSD-4189)

### CODE REVIEW

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/DealRegistrationList/DealRegistrationList.tsx [hunk 1]
```diff
@@ -219,6 +219,7 @@ const DealRegistrationList = () => {
 		{
 			component: (
 				<DateFilter
+					clearInputs={!filters?.dataSubmitted?.dates.startDate?.length && !filters?.dataSubmitted?.dates.endDate?.length}
```
Comment (ryanschuhler) [inline workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/DealRegistrationList/DealRegistrationList.tsx:222]: @eduardo-diniz , would it be possible to do this check in the useEffect() inside of DateFilter.ts? May save us some variable/property setting and passing if we can, and make things a lot simplier and cleaner

Comment (eduardo-diniz) [reply workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/DealRegistrationList/DealRegistrationList.tsx]: Yes, sure
I send `filters?.['previousPropertyDate']` to DateFilter.ts, could you check the PR again pls?

**Review by ryanschuhler — APPROVED** [review]

**Comment by ryanschuhler** [comment]
ci:forward

**Comment by eduardo-diniz** [comment]
ci:test:sf

**Comment by ryanschuhler** [comment]
ci:forward:force

**Comment by eduardo-diniz** [comment]
ci:test:sf

**Comment by ryanschuhler** [comment]
ci:forward:force

**Comment by ryanschuhler** [comment]
thanks! https://github.com/brianchandotcom/liferay-portal/pull/151151

**Pull request closed without merging**